Tasks/VsTest/distributedtest.ts

```diff
@@ -143,12 +143,32 @@ export class DistributedTest {
             throw new Error('Failed to start Modules/DTAExecutionHost.exe.');
         });
 
+        var consolidatedCiData = {
+                    agentFailure: false,
+                    batchingType: models.BatchingType[this.dtaTestConfig.batchingType],
+                    batchsize: this.dtaTestConfig.numberOfTestCasesPerSlice,
+                    codeCoverageEnabled: this.dtaTestConfig.codeCoverageEnabled,
+                    environmenturi: this.dtaTestConfig.dtaEnvironment.environmentUri, 
+                    numberOfAgentsInPhase: this.dtaTestConfig.numberOfAgentsInPhase,
+                    numberOfTestCasesPerSlice: this.dtaTestConfig.numberOfTestCasesPerSlice,
+                    overridesTestRunParameters: utils.Helper.isNullOrUndefined(this.dtaTestConfig.overrideTestrunParameters) ? 'false' : 'true',
+                    pipeline: tl.getVariable('release.releaseUri') != null ? "release" : "build",
+                    runInIsolation: this.dtaTestConfig.runTestsInIsolation,
+                    task: "VsTest",
+                    testSelection: this.dtaTestConfig.testSelection,
+                    usesMultiCoreParallel: this.dtaTestConfig.runInParallel,
+                    usesTia: this.dtaTestConfig.tiaConfig.tiaEnabled,
```

Comment:
please keep it consistent with task properties. as long as information is flowing, the naming should be fine.